tasks/utils/cmd.js

```diff
@@ -0,0 +1,42 @@
+const { isArr, isStr, exists } = require('@keg-hub/jsutils')
```

Comment:
Ohh, I like these. I started a keg-cli-utils repo, but I have not had time to finish it. I'll probably end up adding these to it. They're super helpful